.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "cSpell.words": [
+        "Embedder",
+        "pluggable"
+    ]
+}
```

Comment:
I think this should not be committed and pushed

Reply:
I agree, ignored the .vscode settings folder